src/figuresClasses.ts

```diff
@@ -1,11 +1,68 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (!a || !b || !c || a + b <= c) {
+      throw new Error();
```

Comment:
- [CODE STYLE] - If you throw an error try to provide its detailed description in error message.
![image](https://github.com/mate-academy/ts_figures_classes/assets/63243136/a03454de-15a4-406b-925d-ce23c42d35f6)
- And the condition is really messy, I have no idea what you check here